src/test/resources/original.json

```diff
@@ -0,0 +1,59 @@
+{
+  "_id": "asdas",
+  "_rev": "2-a72e29284ebf401cd7cd8f1aca69af9b",
+  "name": "asdas",
+  "dist-tags": {
+    "latest": "1.0.0"
+  },
+  "versions": {
+    "1.0.0": {
+      "name": "asdas",
+      "version": "1.0.0",
+      "description": "",
+      "main": "1.js",
```

Comment:
Why there is no `1.js` in the project?

Reply:
It's inside tgz-file, we don't need to verify unpacked content. NPM will check sha sums in IT